src/mustache_core.js

```diff
@@ -170,8 +176,11 @@ var core = {
 					var isObserveList = types.isMapLike(finalValue);
 
 					if(isObserveList ? finalValue.attr("length") : finalValue.length) {
```

Comment:
looks like `.attr("length")` might get called here ... but I think this should probably be moved into the helper.